identity-service/src/audiusLibsInstance.js

```diff
@@ -36,7 +41,13 @@ class AudiusLibsWrapper {
         }
       },
       isServer: true,
-      captchaConfig: { serviceKey: config.get('recaptchaServiceKey') }
+      captchaConfig: { serviceKey: config.get('recaptchaServiceKey') },
+      solanaWeb3Config: AudiusLibs.configSolanaWeb3({
+        solanaClusterEndpoint: config.get('solanaEndpoint'),
+        // Never use the relay path in identity
+        shouldUseRelay: false,
+        feePayerSecretKey,
```

Comment:
in that case, `feePayerKeypair` will also be null and Solana relaying will fail. But Identity should still start up properly since libs handles this null case